zabbixprovisioner/provisioner/prometheus_test.go

```diff
@@ -0,0 +1,54 @@
+package provisioner_test
+
+import (
+	"testing"
+
+	"github.com/devopyio/zabbix-alertmanager/zabbixprovisioner/provisioner"
+)
+
+const (
+	rulesOKpath      = "/home/edas/go/src/github.com/devopyio/zabbix-alertmanager/zabbixprovisioner/provisioner/testfiles/rulesOK_test.yml"
```

Comment:
as mentioned befoeruse `testdata` directory instead of `testfiles` and load them using relative path `./testadata/filename.yml`, as this won't work on any other pc other than yours.